about.html

```diff
@@ -71,15 +71,19 @@ <h1 class="display-4">ABOUT ME</h1>
 
                 </div>
                 <div class="col-lg-6 footer__social">
-
+                </a>
```

Comment:
NIepotrzebny znacznik zamknięcia , ponizej tez znajduje się a. Taki zapis to błąd bo masz link w linku.Usuń ten znacznik